tools/Makefile

```diff
@@ -87,3 +94,6 @@ spiffsclean: remove-image
 	rm -f ./spiffsimg/spiffsimg
 	rm -f ./spiffsimg/spiffs.lst
 
+#PDIR := ../$(PDIR)
+#sinclude $(PDIR)Makefile
```

Comment:
I am not sure why you include these two lines, even if commented out.

IMO, the normal reason to include commented out lines is to indicate to developers that they might want to enable these options to add enriched features.  In the case of both the `luac_cross` and `spiffimg` Makefiles these are using the host toolchain and not the xtensa one so doing the `sinclude $(PDIR)Makefile` make file magic would actually break the build in this case.

So please don't add these misleading lines :smile:

Reply:
Yes that's a leftover of my tries to get the int/float detection in the main Makefile.
Removing!

Reply:
Gregor, have you got your head around the recursion magic?  For a standard library, e.g. `modules` Root make→app make→modules make→(sinclude) app make→(sinclude) root make, so for the xtensa toolchain stuff all of the hard rules are in the root make driven by a bunch of variables set in the modules make (or equivalent).  

The host target makes (luac_cross and spiffsimg) are different in that they do the actual (host)compile and build of the component and don't to CBs so no `sinclude` statements: Root make→app make→lua make→lua_cross make.